src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/pkghtml/InputJavadocPackageHtmlIgnored2.java

```diff
@@ -1,7 +1,11 @@
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage.pkghtml;      // ok
+/*
+allowLegacy = "true"
```

Comment:
You need to change Test class also

Reply:
violation lines in the test files have been updated @romani

Reply:
Fixing CI errors